hnn_core/io.py

```diff
@@ -1,341 +0,0 @@
-"""Network io"""
```

Comment:
why was this file deleted? It looks unrelated to the PR

Reply:
Yeah this should have been with PR #704 when it was renamed. Seem like the deletion crept back in when resolving conflicts during the rebase. This PR needs to be reworked so I'll create a separate PR with just the deletion.

Reply:
sorry I should have picked that up during the review of #704 :(